pkg/ctl/create/cluster.go

```diff
@@ -66,7 +65,6 @@ func createClusterCmd(rc *cmdutils.ResourceCmd) {
 
 	rc.FlagSetGroup.InFlagSet("Cluster and nodegroup add-ons", func(fs *pflag.FlagSet) {
 		cmdutils.AddCommonCreateNodeGroupIAMAddonsFlags(fs, ng)
-		fs.BoolVar(&params.addonsStorageClass, "storage-class", true, "if true (default) then a default StorageClass of type gp2 provisioned by EBS will be created")
```

Comment:
Let's make sure we mention this in the changelog

Reply:
Added a note https://github.com/weaveworks/eksctl/pull/992/files#r305223547

Reply:
Thanks ✨